.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+## Checklist
+
+- [ ] If changing `apdb.yaml`, it usually requires version number increment, check [schema versioning page](https://rubinobs.atlassian.net/wiki/external/OGUyOGM1OTY0MTRkNGFmNThkNWM4N2U1NGFmODJkNWY) for details.
```

Comment:
I think @ebellm said that changes to `apdb.yaml` also need to be synchronized with https://github.com/lsst/alert_packet?

Reply:
We can certainly mention that too, please suggest wording for it.

Reply:
> I think @ebellm said that changes to `apdb.yaml` also need to be synchronized with https://github.com/lsst/alert_packet?

I see that there are detailed instructions under https://github.com/lsst/alert_packet/tree/main?tab=readme-ov-file#adding-a-new-schema

We can provide a direct link to this section in the sdm_schemas docs.